.gitmodules

```diff
@@ -7,9 +7,6 @@
 [submodule "test/lib/googletest"]
 	path = test/lib/googletest
 	url = https://github.com/google/googletest.git
-[submodule "examples/cmake_example/lib/cpp-crypto"]
```

Comment:
Why are we removing this?  Doesn't CI need this to build the examples?

Reply:
CI uses the projects root in `~/project`. TMK, it doesn't use `cmake_example` and has been building fine.

Main reason for this is because you have to update that submodule every time you update the Cpp-Crypto repo, otherwise it remains outdated.

It also downloads every time you pull in submodules for the main project.

Reply:
Ok.  I'm cool with that.

A TODO might be to see how we can get CI to built examples too.  Maybe a custom internal project that pulls in the example source files.